binaries/apiserver/main.go

```diff
@@ -10,7 +11,9 @@ func main() {
 	protocolFactory := thrift.NewTBinaryProtocolFactoryDefault()
 	transportFactory := thrift.NewTTransportFactory()
 
-	err := server.Serve(server.NewHandler(), "localhost:9090", transportFactory, protocolFactory)
+	queue, _ := memory.NewSimpleQueue()
+	handler := server.NewHandler(queue)
+	err := server.Serve(handler, "localhost:9090", transportFactory, protocolFactory)
```

Comment:
Probably not, but note that this isn't the binary we run in aurora. For that, we have a binary that mixes in twitter-specific logic.